tjmonopix2/system/tjmonopix2.py

```diff
@@ -531,9 +543,12 @@ def update(self, force=False):
 
 
 class ShiftPatternBase(object):
-    '''
-        Base class for shift patterns
-    '''
+    """Base class for shift patterns
+
+    Parameters
+    ----------
+    object : class
+    """
```

Comment:
The object point is the same as before. Maybe add a bit more what the class does: it specifies the methods that all shift patterns need to implement.